src/components/cards/chefCard/PEChefCard.tsx

```diff
@@ -7,18 +7,18 @@ import PeopleEatIcon from '../../standard/icon/PEIcon';
 
 export interface PEChefCardProps {
     imageUrl?: string;
-    name?: string;
-    location?: string;
-    rank?: string;
+    name: string;
+    location: string;
+    rank: string;
     rating: {
-        average?: number;
-        count?: number;
+        average: number;
+        count: number;
     };
     categories?: string[];
-    menus?: string[];
+    kitchens?: string[];
```

Comment:
Ideally both of these array would not be optiona instead handle the case that they are empty lists

```suggestion
    categories: string[];
    kitchens: string[];
```